server/package.json

```diff
@@ -30,16 +30,20 @@
   },
   "homepage": "https://github.com/nvuillam/vscode-groovy-lint#readme",
   "dependencies": {
-    "debug": "^4.1.1",
-    "find-java-home": "^1.1.0",
-    "fs-extra": "^8.1.0",
-    "glob-promise": "^3.4.0",
-    "npm-groovy-lint": "^10.0.0",
-    "vscode-languageserver": "^6.1.1",
-    "vscode-languageserver-textdocument": "^1.0.1",
-    "vscode-uri": "^2.1.1"
+    "@types/vscode": "^1.85.0",
```

Comment:
shoudn't @types/vscode be in devDependencies ?

Reply:
Quite likely, there was a lot of messing around during the development of this to get things playing nice, need to do a self review and clean up pass.